code/modules/reagents/machinery/chem_master.dm

```diff
@@ -78,6 +79,8 @@ var/global/list/pillIcon2Name = list("oblong purple-pink", "oblong green-white",
 	max_bottle_size = initial(max_bottle_size) + lasercount*5
 	max_pill_count = initial(max_pill_count) + manipcount*5
 	handle_new_reservoir(scancount*25+100)
+	max_pill_size = initial(max_pill_size)+manipcount*25 // i suck at math
```

Comment:
I hate to shine a light on your comment here but I think your math is... off. You're adding the initial value of 50 to 25/50/75

This will result in: T1 = 75, T2 = 100, T3 = 125